scripts/py_matter_idl/matter_idl/test_supported_types.py

```diff
@@ -73,8 +73,8 @@ def testAllTypesSupported(self):
                     "boolean", "single", "double",
                     # handled as specific bitmaps
                     "bitmap8", "bitmap16", "bitmap24", "bitmap32", "bitmap64",
```

Comment:
As a side note, bitmap24 is also not described in the DM section, so we could consider removing that as well?

Reply:
Based on matter files, indeed, we do not seem to be using it at all:

```
 ❯ rg 'bitmap\d+' --glob '*.matter' | sd '.*(bitmap\d*).*' '$1' | sort | uniq -c
    273 bitmap16
   2990 bitmap32
     10 bitmap64
   1205 bitmap8
```

third_party/openthread/ot-efr32/third_party/silabs/gecko_sdk/app/zcl/types.xml used to have it it seems. Will remove them as well.

Reply:
Interestingly enough, the type "bitmapX" does not seem to be defined at all, only "mapX". It would be nice if types were unified, however apparently they are not. For now I will just remove the non-standard sized bitmaps.